app/importers/envirotech/relational_data.rb

```diff
@@ -0,0 +1,111 @@
+module Envirotech
+  class RelationalData
+    cattr_accessor :relations, :solution_ids_names
+
+    def import
+      return unless  RelationalData.relations.present?
+      process_solution_relations
+      process_regulation_relations
+    end
+
+    private
+
+    def process_regulation_relations
+      regulations = Envirotech::Consolidated.search_for(sources: 'regulations', size: 100)
```

Comment:
Throughout our discussion I've been surprised that you start the matching of related documents by fetching a list of documents from ES. I was expecting that you'd start by iterating over the fetched JSON, like in this pseudo-code:
```
json = Envirotech::ToolkitData.fetch_relational_data
json.keys.each do |issue_name|
  issue = ES.fetch(issue with name "issue_name")
  json[issue_name].each do |regulation_name|
    regulation = ES.fetch(regulation with name "regulation_name")
    issue.regulation_ids << regulation.id
    regulation.issue_ids << issue.id
    json[issue_name][regulation_name].each do |solution_name|
      solution = ES.fetch(solution with name "solution_name")
      regulation.solution_ids << solution.id
      solution.regulation_ids << regulation.id
      solution.save
    end
    regulation.save
  end
  issue.save
end
```
Doing this avoids the problem when there are more than 100 issues or regulations. Do you think iterating over the JSON is better, or is there a problem doing so that I'm not seeing?